packages/server/src/definitions/openapi.ts

```diff
@@ -9,273 +9,273 @@ export interface paths {
       parameters: {
         header: {
           /** The ID of the app which this request is targeting. */
-          "x-budibase-app-id": components["parameters"]["appId"];
-        };
-      };
+          "x-budibase-app-id": components["parameters"]["appId"]
```

Comment:
@mike12345567 not sure why these changes have come in, will this break anything?

Reply:
Looks like Prettier has got its hands on the openAPI Typescript definitions again, they are supposed to be excluded from prettier but it seems it has picked them up somehow!

In terms of will it break anything, no it doesn't cause any issues, in terms of usage it should be identical - its just non-ideal as its a generated file, each time it is generated again prettier will then need to be ran again etc.

Reply:
Figured this one out, it was happening because we were doing an additional es-lint inside server and worker for typescript files, and in these cases we weren't re-ignoring the openapi.ts file. 

I've moved everything to the top level so we don't need to keep multiple sets of ignore / config files in future, see this commit:
https://github.com/Budibase/budibase/pull/5044/commits/70c54e983d6e3645d14aa200091a5cc01b08aa84

I noticed that some lint violations were being detected this time, which seems to be a result of the previous eslint configs in worker/server not doing anything due to the `files` array being empty. For now I have turned the violations off in the eslint typescript config to reduce noise here, and I'll turn them back on and address in a follow up pr.